catalogs/catalog-jdbc-postgresql/src/test/java/com/datastrato/gravitino/catalog/postgresql/integration/test/TestMultipleJDBCLoad.java

```diff
@@ -115,8 +112,10 @@ public void testCreateMultipleJdbc() throws URISyntaxException {
     mysqlConf.put(
         JdbcConfig.JDBC_URL.getKey(),
         StringUtils.substring(
-            mySQLContainer.getJdbcUrl(), 0, mySQLContainer.getJdbcUrl().lastIndexOf("/")));
-    mysqlConf.put(JdbcConfig.JDBC_DRIVER.getKey(), mySQLContainer.getDriverClassName());
+            mySQLContainer.getJdbcUrl(TEST_DB_NAME),
+            0,
+            mySQLContainer.getJdbcUrl(TEST_DB_NAME).lastIndexOf("/")));
```

Comment:
Can this be simplified here?

Reply:
the JdbcUrl here don't need database name, so I plan to add an overloaded `getJdbcUrl` method:

```
  public String getJdbcUrl() {
    return format("jdbc:mysql://%s:%d", getContainerIpAddress(), MYSQL_PORT);
  }
```

And I noticed that `jdbcUrl` in `pgConf` should also be the one without datebase name, I'll fix that.

Reply:
If doing so can simplify the code, then it is advisable to add it.

Reply:
Actually, jdbcUrl of PostgreSQL here should contain database name, otherwise, test will fail.
https://github.com/datastrato/gravitino/actions/runs/8994207322/job/24707359899?pr=3195#step:9:2068

Reply:
Maybe you can add a method named `getJdbcUrlWithDb(String database)` ?

Reply:
I think overloaded `getJdbcUrl` with documentation is informative enough,
but I'm ok with `getJdbcUrlWithDb(TestDatabaseName testDatabaseName)` and `getJdbcUrl()`